### PR TITLE
[DICOM Archive - good] updated module help text

### PR DESCRIPTION
**Pull request by knoll-alex**
This is a re-do of PR #5621 in attempt to remove an extraneous file from that PR (couldn't revert commit). PR 5621 should be closed once this is approved.

#I## Testing instructions

1. Ensure there are no extraneous files

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576
* https://github.com/aces/Loris/pull/5621

image

### CODE REVIEW

File: modules/dicom_archive/help/dicom_archive.md [hunk 1]
```diff
@@ -1,17 +1,9 @@
 # Dicom Archive
 
-The Dicom Archive module displays all scans that have been uploaded
-to LORIS, whether or not they have been inserted or not (due to protocol
-violations, mislabeling, not having been run, etc.) It's a filterable archive
-of every single dicom that this LORIS instance knows about.
+This module displays all scans that have been uploaded to LORIS. This includes scans that have been inserted, as well as those that haven’t (due to protocol violations, mislabeling, not having been run, etc.) In other words, it's an archive of every single DICOM that your LORIS instance knows about.
```
Comment (johnsaigle) [inline modules/dicom_archive/help/dicom_archive.md]:
```suggestion
This module displays all scans that have been uploaded to LORIS. This includes scans that have been inserted, as well as those that were rejected (due to protocol violations, mislabeling, not having been run, etc.) In other words, it's an archive of every single DICOM that your LORIS instance knows about.
```

I'm not sure if 'rejected' is the right word here but I think it's a bit more clear

Comment (christinerogers) [reply modules/dicom_archive/help/dicom_archive.md]: done. kindly re-review @johnsaigle

**Comment by christinerogers** [comment]
@driusan  this is ready to merge.

File: modules/dicom_archive/help/dicom_archive.md [hunk 2]
```diff
@@ -1,17 +1,9 @@
...
-Clicking on the "View Details" link in the Metadata column will take you
-to a page displaying more details of every scan and clicking on "View Images"
-in the MRI Browser column will take you to the session containing those
-images in the LORIS imaging browser.
+In the resulting table, you might notice some clickable cells. In the *Metadata* column, click **View Details** to see the compressed package of DICOM files for that scan, along with their metadata. In the *MRI Browser* column, click on **View Images** to visualize and QC these scans in the Imaging Browser module.
 
-Clicking on the "Archive Location" link will begin a download of the scans'
-DICOMs.
\ No newline at end of file
+In the **Archive Location** column, click any link to download the DICOMs from all scans in this session.
```
Comment (driusan) [inline modules/dicom_archive/help/dicom_archive.md]:
```suggestion
In the **Archive Location** column, click a link to download the DICOMs from all scans in that session.
```

I find the current wording a little awkward and hard to parse

**Comment by christinerogers** [comment]
Alex/Dave -- changes made seem good to go.  can we get a re-review, maybe @johnsaigle  
and then a merge? @driusan
